File: Logica de Programação/Venda.java
import java.util.Scanner;

public class Venda {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double venda, desc, val_final;

        System.out.print("Digite o valor da venda: ");
        venda = entrada.nextDouble();

        if (venda >= 300) {
            System.out.print("Digite o valor do descont: ");
            desc = entrada.nextDouble();

            val_final = venda - (venda  * (desc/100));
            System.out.println("O valor final de sua compra é de R$" + val_final);
        }

        else {
        System.out.println("O valor final de sua compra é de R$" + venda);
        }
        
        entrada.close();
 }
}

File: Logica de Programação/Venda.py
venda = float(input("Digite o valor da venda: "))

if venda >= 300:
    desconto = float(input("Digite o avalor do desconto: "))
    val_final = venda - (venda * (desconto/100))
    print(f"O valor final da sua compra é de R${val_final}")

else:
    print(f"O valor final de sua compra é de R${venda}")